word_language_model/main.py

```diff
@@ -217,7 +227,9 @@ def export_onnx(path, batch_size, seq_len):
     model = torch.load(f)
     # after load the rnn params are not a continuous chunk of memory
     # this makes them a continuous chunk, and will speed up forward pass
-    model.rnn.flatten_parameters()
+    # Currently, only rnn model supports flatten_parameters function.
+    if args.model != 'Transformer':
```

Comment:
I'd check explicitly for the models you want to allow this for. If someone ads a model this will cause a bug. It's a minor thing, but I think good practice.